admin.go

```diff
@@ -442,11 +454,23 @@ func (ca *clusterAdmin) DescribeConfig(resource ConfigResource) ([]ConfigEntry,
 		Resources: resources,
 	}
 
-	b, err := ca.Controller()
+	var (
+		b   *Broker
+		err error
+	)
+
+	// DescribeConfig of broker must be sent to the broker in question
+	if resource.Type == BrokerResource {
+		id, _ := strconv.Atoi(resource.Name)
+		b, err = ca.findBroker(int32(id))
+	} else {
+		b, err = ca.Controller()
```

Comment:
When describing a non-broker resource, the request can be sent to any broker, so we could use `findAnyBroker()` here.

Reply:
👍